pkg/chain/evmregistry.go

```diff
@@ -17,18 +17,7 @@ import (
 	"github.com/smartcontractkit/ocr2keepers/pkg/types"
 )
 
-const (
-	ActiveUpkeepIDBatchSize int64  = 10000
-	separator               string = "|"
-)
-
```

Comment:
is this strange strictly needed? it looks safe as just a move, but trying to minimise the changes in bugfix

Reply:
No. But like the client, the evm registry isn't used by the chainlink node. It is only used by the simulator.